certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -3985,418 +4992,1389 @@
+                "type": "string",
+                "description": "Event ID.",
+                "title": "Event ID"
+            },
+            "eventCode": {
+                "type": "string",
+                "description": "Event Code.",
+                "title": "Event Code"
+            },
+            "eventName": {
+                "type": "string",
+                "description": "Event Name.",
+                "title": "Event Name"
+            },
+            "eventStartDate": {
+                "type": "string",
+                "description": "Event Start Date.",
+                "title": "Event Start Date"
+            },
+            "eventStartTime": {
+                "type": "string",
+                "description": "Event Start Time.",
+                "title": "Event Start Time"
+            },
+            "eventEndDate": {
+                "type": "string",
+                "description": "Event End Date.",
+                "title": "Event End Date"
+            },
+            "eventEndTime": {
+                "type": "string",
+                "description": "Event End Time.",
+                "title": "Event End Time"
+            },
+            "eventTimezone": {
+                "type": "string",
+                "description": "Event Time Zone.",
+                "title": "Event Time Zone"
+            },
+            "eventDescription": {
+                "type": "string",
+                "description": "Event Description.",
+                "title": "Event Description"
+            },
+            "shortDescription": {
+                "type": "string",
+                "description": "Short Description.",
+                "title": "Short Description"
+            },
+            "publicEvent": {
+                "type": "boolean",
+                "description": "Public Event.",
+                "title": "Public Event",
+                "enum": [
+                    true,
+                    false
+                ]
+            },
+            "imageUrl": {
+                "type": "array",
+                "items": {
+                    "type": "string"
                 },
-                "callBackUrl": {
-                  "type": "string",
-                  "description": "Callback URL.",
-                  "title": "Callback URL",
-                  "x-ms-visibility": "internal",
-                  "x-ms-notification-url": true
+                "description": "imageUrl"
+            },
+            "externalUrl": {
+                "type": "string",
+                "description": "External Url.",
+                "title": "External Url"
+            },
+            "externalCode": {
+                "type": "string",
+                "description": "External Code.",
+                "title": "External Code"
+            },
+            "eventLocations": {
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
```

Comment:
Please add `Event Locations -` prefix to these properties
Also add missing `title`s